meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -414,30 +414,25 @@ BOOTSCHEME_LABELS:stm32mp1common += "optee"
 BOOTSCHEME_LABELS:stm32mp1common:remove = "trusted"
 UBOOT_CONFIG:remove:stm32mp1common = "basic_stm32mp15"
 UBOOT_SUFFIX:stm32mp1common = "bin"
+ST_BOOTFS:stm32mp1common = "0"
+ST_VENDORFS:stm32mp1common = "0"
+ST_USERFS:stm32mp1common = "0"
+M4_INSTALLDIR:stm32mp1common = "${prefix}"
+UBOOT_LOADADDRESS:stm32mp1common = "0xC2000000"
+UBOOT_ENTRYPOINT:stm32mp1common = "0xC2000000"
+UBOOT_DTB_LOADADDRESS:stm32mp1common = "0xC4000000"
 ## stm32mp15-eval
 LINUX_A7_EXAMPLES_DT:remove:stm32mp15-eval = "stm32mp157c-ev1-a7-examples"
 CUBE_M4_EXAMPLES_DT:remove:stm32mp15-eval = "stm32mp157c-ev1-m4-examples"
 UBOOT_DEVICETREE:stm32mp15-eval = "stm32mp157c-ev1"
 IMAGE_FSTYPES:remove:stm32mp15-eval = "wic wic.gz wic.bmap"
 EXTRA_IMAGEDEPENDS:append:stm32mp15-eval = "flashlayouts-stm32mp1"
-UBOOT_LOADADDRESS:stm32mp15-eval = "0xC2000000"
-UBOOT_ENTRYPOINT:stm32mp15-eval = "0xC2000000"
-UBOOT_DTB_LOADADDRESS:stm32mp15-eval = "0xC4000000"
-UBOOT_RD_LOADADDRESS:stm32mp15-eval = "0xC4400000"
-UBOOT_RD_ENTRYPOINT:stm32mp15-eval = "0xC4400000"
```

Comment:
I don't see UBOOT_RD* variables for the target common. They are only being removed. I will assume this is on purpose

Reply:
Yes, they are not needed.